jarviscli/plugins/MipsConverter.py

```diff
@@ -0,0 +1,534 @@
+import os
+from plugin import plugin
+
+FILE_PATH = os.path.abspath(os.path.dirname(__file__))
+
+
+@plugin("mips")
+class MipsConverter:
+    """
+    Jarvis will convert any Assembly(mips) statement for you
+    from assembly to machine code or from machine code(hex)
+    to Assembly.
+
+    Just write your command as follows:
+    For Hex to Assembly:
+        mips XXXXXXXX
+        The above command calls mips with a 8 digit Hex code
+        which makes a 32 bit instruction where the X are the
+        Hex digits.
+
+    For Assembly to Hex:
+        mips Addi $t2, $t1, 0x12
+        The above command calls mips with any mips basic instruction,
+        in this case ADDI. There should be a space between the
+        instruction keyword and the first register and then the
+        rest can be separated with commas.
+
+    A good practice would be to give a space between any separate part of
+    the instruction such as between keyword and register, or keyword
+    and immediate, or register and immediate.
+    """
+    def __call__(self, jarvis, s):
+        self.__init__()
```

Comment:
Almost forgot: This line is obsolete, because __init__() is called on class construction and does not need be called manually.

> So if it is indeed necessary to change these two custom methods with python ones, I would prefer it this way. 

Ok, then we'll keep the methods like you prefer.